src/context/ProjectContextProvider.tsx

```diff
@@ -17,13 +16,15 @@ interface ProjectContextValue {
   appName: string;
   projectId: string;
   projectIdOrName: string;
+  isLoading: boolean;
 }
 
 export const ProjectContext = createContext<ProjectContextValue>({
   project: null,
   appName: '',
   projectId: '',
   projectIdOrName: '',
+  isLoading: false,
```

Comment:
shouldn't the default value for `isLoading` be true?

Reply:
switched.